### PR TITLE
Allow supplying sources from the command line

### PR DESCRIPTION
**Pull request by MichelKansou**
Added parsing sources in command line feature #1 and updated command lines in yaml

### CODE REVIEW

**Comment by MichelKansou** [comment]
@msfjarvis I think this will do the job

File: src/main.rs [hunk 1]
```diff
@@ -74,3 +53,37 @@ fn main() {
         println!("{}", contents);
     }
 }
+
+fn parse_file(filepath: &str) -> HashMap<String, Source> {
+    println!("Sources path {}", filepath);
+    let mut list: HashMap<String, Source> = HashMap::new();
+    if filepath != "" {
+        let file = File::open(filepath).unwrap_or_else(|err| {
+            eprintln!("Problem openning file: {}", err);
```
Comment (msfjarvis) [inline src/main.rs]:
```suggestion
            eprintln!("Problem opening file: {}", err);
```

File: src/main.rs [hunk 2]
```diff
@@ -74,3 +53,37 @@ fn main() {
...
+            process::exit(1);
+        });
+
+        let lines = BufReader::new(file).lines();
+        for line in lines {
+            let content = line.unwrap_or_else(|err| {
+                eprintln!("Problem parsing lines: {}", err);
```
Comment (msfjarvis) [inline src/main.rs]:
```suggestion
                eprintln!("Problem parsing sources file: {}", err);
```

**Comment by MichelKansou** [comment]
@msfjarvis your welcome 😁

**Pull request merged**